install/jaxrs/bin/ts.jte

```diff
@@ -77,20 +75,18 @@ javatest.timeout.factor=1
 ###################################################################
 ###################################################################
 ###################################################################
-##  CI SPECIFIC PROPERTIES LIVE BELOW
+##  RI SPECIFIC PROPERTIES LIVE BELOW
 ###################################################################
 ###################################################################
 ###################################################################
 
 ###################################################################
-## When installing Jakarta RESTful Web Services TCK/CI on Windows, 
-## users must install Jakarta Restful Web Services TCK and
-## the CI on the same drive.  Also note that you should never
+## When installing JAX-RS TCK/RI on Windows, users must install JAXRS-TCK and
```

Comment:
looks like we changed Jakarta RESTFul Web Services reference back to JAX-RS.  We should keep the Jakarta RESTFul Web Services

Reply:
For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.